chart/README.md

```diff
@@ -514,18 +516,19 @@ Some example commands are shown below.
    --set ingress.ingressClassName=alb
    --set ingress.annotations."alb\.ingress\.kubernetes\.io/scheme"="internet-facing"
    --set ingress.annotations."alb\.ingress\.kubernetes\.io/healthcheck-path"="/ping" 
-   .
+   sonatype/nexus-iq-server-ha --version <version>
    ```
 
 #### External Database, Dynamic EFS, Dynamic ALB, and Secrets
    ```
-   helm install mycluster
-   --set iq_server.serviceAccountName="nxlc-default-d95tfx8l-service"
+   helm repo add sonatype https://sonatype.github.io/helm3-charts/
```

Comment:
minor: do we really need to repeat this repo adding instruction with every helm install? its kind of implicit when you use `sonatype/nexus-iq-server-ha --version xxx` and we can just have once mentioned somewhere at the beginning.

Reply:
Fixed.